### PR TITLE
I finished changes in the images screen.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
### Changes have been made to several pages.

  * I added the possibility to delete the user from the list.
 
Screenshot from 2023-06-08 17-55-17
Screenshot from 2023-06-08 17-59-47

Screenshot from 2023-06-08 18-04-32

### CODE REVIEW

File: screens/MessagesUsers/index.js [hunk 1]
```diff
@@ -28,6 +36,42 @@ const MessagesUsers = ({navigation}) => {
     return () => clearTimeout(timer);
   }, []);
 
+  // eslint-disable-next-line react/no-unstable-nested-components
```
Comment (M7Arman) [inline screens/MessagesUsers/index.js:39]: this is not ok, fix it

File: screens/MessagesUsers/index.js [hunk 2]
```diff
@@ -28,6 +36,42 @@ const MessagesUsers = ({navigation}) => {
...
+          </Pressable>
+        </View>
+      </View>
+    );
+  };
+
+  useEffect(() => {
+    setModalVisible(false);
+    if (isAnswer === 'YES') {
+      console.log(removeId);
```
Comment (M7Arman) [inline screens/MessagesUsers/index.js:57]: remove

File: screens/MessagesUsers/index.js [hunk 3]
```diff
@@ -28,6 +36,42 @@ const MessagesUsers = ({navigation}) => {
...
+      const friends = data.filter(item => item.id !== removeId);
+      setData(friends);
+    }
+    setModalVisible(false);
+    return () => setAnswer('');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (M7Arman) [inline screens/MessagesUsers/index.js:63]: fix

File: screens/MessagesUsers/index.js [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
+import SwipeableFlatList from 'react-native-swipeable-list';
+import {theme} from '../../assets/theme/theme';
+import PermissionModal from '../../components/Permission/Modal';
+import UsersMessagesModal from '../../components/Permission/children/remove';
 import styles from './style';
 
 const MessagesUsers = ({navigation}) => {
   const [loading, setLoading] = useState(true);
   const [data, setData] = useState([]);
   const [state, setState] = useState(data);
```
Comment (M7Arman) [inline screens/MessagesUsers/index.js:19]: state is not good name

Comment (M7Arman) [reply screens/MessagesUsers/index.js:19]: also data state can't be default value for another state

File: screens/MessagesUsers/index.js (hunk 1 above)
Comment (MiqayelChaloyan) [reply screens/MessagesUsers/index.js:39]: I will think of a better solution.

File: screens/MessagesUsers/index.js (hunk 4 above)
Comment (MiqayelChaloyan) [reply screens/MessagesUsers/index.js:19]: I will fix it.

File: screens/MessagesUsers/index.js (hunk 2 above)
Comment (MiqayelChaloyan) [reply screens/MessagesUsers/index.js:57]: I also changed it to a more understandable name here.

Comment (MiqayelChaloyan) [reply screens/MessagesUsers/index.js:57]: For the entire code, I will think of a more correct solution and remove all eslint related comments.

**Pull request merged**